runtime/src/bank.rs

```diff
@@ -4447,6 +4452,7 @@ impl Bank {
                         LoadedProgramType::FailedVerification,
                     ))
                 });
+                program.tx_usage_counter.store(*count, Ordering::Relaxed);
```

Comment:
Shouldn't we add the number instead of overwriting the counter?

Reply:
`load_program()` creates a new instance of `LoadedProgram` which has `tx_usage_counter` as 0. So adding the number will be same as overwriting here.
We add the number later in the call to `replenish()`, in case there was an unloaded entry in the cache.